bash/t/terraform/terraform_run.functions

```diff
@@ -48,7 +48,7 @@ setup_terraform_run_stubs() {
 
     stub_no_unpushed_changes="$(echo -e "no_unpushed_changes() {\nreturn $nrc\n}")"
     eval "$stub_no_unpushed_changes"
-    
+
```

Comment:
👍 I HATE stooopid,redundant whitespace ...

Reply:
It's not me, it's the [Atom](https://atom.io) :)